src/BizHawk.Client.Common/config/Config.cs

```diff
@@ -255,6 +255,12 @@ public void ResolveDefaults()
 		public string SoundDevice { get; set; } = "";
 		public int SoundBufferSizeMs { get; set; } = 100;
 
+		public bool MuteInBG { get; set; } = false;
+
+		public bool MuteOnLag { get; set; } = false;
+
+		public int FPSThreshold { get; set; } = 56;

```

Comment:
Needs to be a percentage; not all consoles run at 60 fps.

Reply:
Just changed it now. Should I make it so the base FPS changes depending on the console? Because I'm not sure how to do that really. I still have 60fps hardcoded in for now

Reply:
You can get the loaded system's nominal framerate as a scalar like this:
https://github.com/TASEmulators/BizHawk/blob/5bf9d387b4fe342a9e9102e6a61cf3379affe7cb/src/BizHawk.Client.EmuHawk/MainForm.Events.cs#L953
(As the comment suggests, there's no helper method, so you can just copy-paste that. You probably don't even need to cache the result.)

Reply:
Should be fixed now, thanks for the help! And sorry for the delay, I'm very busy with school this week